AirportDepartures.playground/Contents.swift

```diff
@@ -16,6 +16,103 @@ import UIKit
 //: e. Use a `String?` for the Terminal, since it may not be set yet (i.e.: waiting to arrive on time)
 //:
 //: f. Use a class to represent a `DepartureBoard` with a list of departure flights, and the current airport
+enum FlightStatus {
+    case EnRoute
+    case Schedule
+    case Canceled
+    case Delayed
+    case Landed
+    case Boarding
+}
+
+struct Airport {
+    var destination: String
+}
+
+struct Flight {
+    let airport: Airport
+    var airline: String
+    var flightNumber: String
+    var departureTime: Date?
+    var terminal: String?
+    var fightStatus: FlightStatus
+}
+
+class DepartureBoard {
+    var departureFlights: [Flight]
+    var currentAirport: String
+    
+    init(departureFlights: [Flight], currentAirport: String) {
+        self.departureFlights = departureFlights
+        self.currentAirport = currentAirport
+    }
+    
+    
+        func passengerAlerts() {
```

Comment:
Not a big deal, but this function has an extra level of indentation that isn't needed. The further you can keep stuff to the left, the less chance you have of line wrapping.